projects/helper/near.js

```diff
@@ -70,10 +69,10 @@ async function sumSingleBalance(balances, token, balance) {
       balances[name] = BigNumber(0)
 
     balances[name] = balances[name].plus(balance)
-    return
+    return balances
   }
 
-  sdk.util.sumSingleBalance(balances, transformAddress(token), balance)
+  await sdk.util.sumSingleBalance(balances, transformAddress(token), balance)
```

Comment:
sdk.util.sumSingleBalance doesn't return a promise, there's not point in awaiting it afaik?

Reply:
> sdk.util.sumSingleBalance doesn't return a promise, there's not point in awaiting it afaik?

that was my suggestion on the last PR oops

Reply:
Yeah, then I looked into `bitBTC.js` and thought it was async after all, fixed it now.

I am seeing the same issue in 16 other projects, can I clean it up or create ticket and leave it for later?

Reply:
if you could clean them up that'd be awesome. I guess they're not doing much harm at the moment so can do it on another PR